### DISCUSSION TITLE
Welcome to ML4W Dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eljejer** [comment]
Thank you @mylinuxforwork for your kind welcome post

**My introduction:**
Retired guy born in 1958 in the Netherlands
After some doubtful professional choices landed at last in IT as a trainee Cobol software engineer in 1986
20 years of programming experience
Switched from Windows to Linux in 2020
Lots of distro hopping and now settled for Hyprland on Manjaro Linux
Planning to become an active community member here; learning, educating and maybe some coding

**Comment by kerbyyy** [comment]
Hi, everyone! I'm really glad to be part of this community.

I'm a college IT Major student from the Philippines, born in 2004. Programming and hardware has been my passion ever since, and would really love to land an IT Job soon. For any IT Professionals out there, I would love to hear your stories and maybe some tips as well on what I would face on that industry.

I switched to Linux this 2023. Well, not fully switched. I usually dual boot Windows and Linux, since some of my stuff only works on Windows. I started with Linux Mint, then switched to LMDE, then tried some other distros like Fedora but really hated them. After finally taking effort on learning stuff, I finally settled with Arch Linux.

My first Desktop Environment with Arch was Cinnamon since I got used to Linux Mint. I discovered Hyprland and fell in love with it immediately but failed multiple times implementing it until I found ML4W. I've been using the dotfiles since version 2.6.1 and never switched to anything else ever since. Furthermore, I'm planning to expand my knowledge and maybe be able to contribute to this community as well.

**Comment by Dtress** [comment]
Hello. I am a 54 old guy, final linux's user since about 2002 but not working in IT technology at all. I like to discover DE.
